masonry/README.md

```diff
@@ -12,12 +8,18 @@
 [![GitHub Actions CI status.](https://img.shields.io/github/actions/workflow/status/linebender/xilem/ci.yml?logo=github&label=CI)](https://github.com/linebender/xilem/actions)
 [![Dependency staleness status.](https://deps.rs/crate/masonry/latest/status.svg)](https://deps.rs/crate/masonry)
 
+# Masonry
+
 </div>
 
-Masonry gives you a platform to create a window (using [winit] as a backend) with a tree of widgets.
-It also gives you tools to inspect that widget tree at runtime, write unit tests on it, and generally have an easier time debugging and maintaining your app.
+<!-- cargo-rdme start -->
+
+**A foundational framework for Rust GUI libraries.**
```

Comment:
My personal preference is for this line to also be above the badges. That does require moving the badges into the lib.rs documentation, but I think that's fine.

Either way, I think this should still be centered